.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.0'
+          - 'lts'
```

Comment:
```suggestion
          - 'min'
          - 'lts'
```

Reply:
When 'min' and 'lts' are the same, will it repeat the tests?

Reply:
It will. But they are not the same currently.